src/main/java/org/mastodon/views/trackscheme/display/EditFocusVertexLabelAction.java

```diff
@@ -180,6 +180,12 @@ public void updateHeaderSize( final int width, final int height )
 		headerHeight = height;
 	}
 
+	protected void changeLabel( TrackSchemeVertex vertex, String label )
+	{
+		vertex.setLabel( label );
+		undoPointMarker.setUndoPoint();
```

Comment:
We probably should make an undo point only once, after the whole branch has been renamed. 
Here we store an undo point for every vertex being renamed.